tests/expand/js14-in.jsonld

```diff
@@ -0,0 +1,12 @@
+{
+  "@context": {
+    "@version": 1.1,
```

Comment:
Do we need an explicit version here?

Reply:
I was matching the style of the other js* tests.